pkg/agent/flexvolume/server.go

```diff
@@ -63,7 +63,9 @@ func NewFlexvolumeServer(context *clusterd.Context, volumeAttachmentClient rest.
 
 // Start configures the flexvolume driver on the host and starts the unix domain socket server to communicate with the driver
 func (s *FlexvolumeServer) Start() error {
-	err := configureFlexVolume(flexvolumeDriverFileName, flexVolumeDriverDir)
+
+	driverFile := path.Join("/usr/local/bin/", flexvolumeDriverFileName) // /usr/local/bin/rookflex
```

Comment:
can you please make `/usr/local/bin` a constant too.

Reply:
Ok

Reply:
Ok